miio/device.py

```diff
@@ -249,10 +249,10 @@ def test_properties(self, properties):
         """Helper to test device properties."""
 
         def ok(x):
-            click.echo(click.style(x, fg="green", bold=True))
+            click.echo(click.style(str(x), fg="green", bold=True))
```

Comment:
Are the changes to this file necessary? If yes, could you please open a separate PR as it's not related to the water heater itself?

Reply:
Done https://github.com/rytilahti/python-miio/pull/1180